pkg/utils/utils_unit_test.go

```diff
@@ -980,12 +978,34 @@ func TestRetrieveVcConfigSecret(t *testing.T) {
 	formatter.FullTimestamp = true
 	logger.SetFormatter(formatter)
 	logger.SetLevel(logrus.DebugLevel)
+	confData := `[Global]
+	cluster-id = "cluster1"
+	
+	[VirtualCenter "10.182.1.133"]
+	user = "user@vsphere.local"
+	password = "password"
+	port = "443"`
+	mockSecret := &corev1.Secret{
+		ObjectMeta: metav1.ObjectMeta{Name: "velero-vsphere-config-secret"},
+		Data:       map[string][]byte{"csi-vsphere.conf": []byte(confData)},
+	}
 	tests := []struct {
 		name        string
 		runtimeObjs []runtime.Object
 		config      *rest.Config
 		expectError bool
 	}{
+		{
+			name: "Test secret retrieval",
+			runtimeObjs: []runtime.Object{
+				decoupleVSphereCSIDriverFeatureDisabled,
+				csi201VanillaDeployment,
+				csi201VSphereCredentialSecret,
+				mockSecret,
```

Comment:
Why was decoupleVSphereCSIDriverFeatureDisabled, csi201VanillaDeployment, csi201VSphereCredentialSecret chosen, could you reason it out? 
PS: I don't remember much about these objs, I am sure you have a good reason for picking them up specifically.

Reply:
Since `RetrieveVcConfigSecret` is the caller of `ParseConfig`, I used those objects to reach the path that would allow for the `mockSecret` to be parsed. `RetrieveVcConfigSecret` needed a set of objects to reach the point of calling `ParseConfig`, although the selection of these specific objects is not the only path to reach the call of `ParseConfig`.